docs/debug_virt_stack/logging.md

```diff
@@ -0,0 +1,81 @@
+# Control libvirt logging for each component
+
+Generally, cluster admins can control the log verbosity of each KubeVirt component in KubeVirt CR. For more details, please, check the [KubeVirt documentation](https://kubevirt.io/user-guide/operations/debug/#log-verbosity).
+
+Nonetheless, regular users can also adjust the qemu component logging to have a more fine control over it. The annotation `kubevirt.io/libvirt-log-filters` enables you to modify each component's log level.
+
+Example:
+```yaml
+apiVersion: kubevirt.io/v1
+kind: VirtualMachineInstance
+metadata:
+  annotations:
+    kubevirt.io/libvirt-log-filters: "2:qemu.qemu_monitor 3:*"
+  labels:
+    special: vmi-debug-tools
+  name: vmi-debug-tools
+spec:
+  domain:
+    devices:
+      disks:
+      - disk:
+          bus: virtio
+        name: containerdisk
+      - disk:
+          bus: virtio
+        name: cloudinitdisk
+      rng: {}
+    resources:
+      requests:
+        memory: 1024M
+  volumes:
+  - containerDisk:
+      image: registry:5000/kubevirt/fedora-with-test-tooling-container-disk:devel
+    name: containerdisk
+  - cloudInitNoCloud:
+      userData: |-
+        #cloud-config
+        password: fedora
+        chpasswd: { expire: False }
+    name: cloudinitdisk
+```
+
+Then, it is possible to obtain the logs from the virt-launcher output:
+
+```bash
+$ kubectl  get po
```

Comment:
Nice shortcut, that'll save me 2 keystrokes many times a day :)
However, the full version might look better in docs?
```suggestion
$ kubectl get pods
```